docs/frameworks/fastifyjs.md

```diff
@@ -0,0 +1,356 @@
+---
+description: Learn how to deploy a Fastify.js application with Genezio.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Fastify.js
+
+<head>
+    <title>Fastify.js | Genezio Documentation</title>
+</head>
+
+Fastify.js is a popular Node.js web application framework that simplifies the development of server-side applications. It provides a robust set of features for building web servers and APIs.
+
+<!-- TODO: change to fastify template -->
+
+:::tip
+Get started in no time with the [Fastify.js template](https://github.com/Genez-io/fastify-getting-started).
+:::
+
+# Deployment
+
+Learn how to deploy an existing Fastify.js app using Genezio, a serverless deployment platform that simplifies app management and reduces costs
+
+## Prerequisites
+
+### 1. Install genezio
+
+Use your preferred package manager to install Genezio:
+
+<Tabs>
+  <TabItem className="tab-item" value="npm" label="npm">
+<div id="step1-install-npm">
+  ```
+  npm install genezio -g
+  ```
+  </div>
+  </TabItem>
+  <TabItem className="tab-item" value="pnpm" label="pnpm">
+  <div id="step1-install-pnpm">
+  ```
+  pnpm add -g genezio
+  ```
+  </div>
+  </TabItem>
+  <TabItem  className="tab-item" value="yarn" label="yarn">
+  <div id="step1-install-yarn">
+  ```
+  yarn add global genezio
+  ```
+  </div>
+  </TabItem>
+</Tabs>
+
+### 2. Ensure you have an Fastify.js App
+
+If you don't have an Fastify.js app, you can create one using the following steps:
+
+<details>
+  <summary>**Create a Hello World Fastify.js App**</summary>
+
+### 1. Initialize a New Node.js Project
```

Comment:
```suggestion
<h3> 1. Initialize a New Node.js Project </h3>
```

Reply:
This looks a bit weird in the table of contents on the right side because docusaurus doesn't detect that this is an expandable section.

To fix this we should hide this headers from the ToC.

I researched a bit and we can do that by [changing the markdown headers (`###`) to html header tags.](https://docusaurus.io/docs/next/markdown-features/toc#customizing-table-of-contents-generation)

Please to this change for all the headers in the expandable section.

<img width="297" alt="image" src="https://github.com/user-attachments/assets/e5273187-430e-408c-abca-f411a52c9686">